src/soilbiogeochem/SoilBiogeochemDecompCascadeMIMICSMod.F90

```diff
@@ -925,92 +926,77 @@ subroutine decomp_rates_mimics(bounds, num_soilc, filter_soilc, &
-            ! TODO Excluded spinup terms for now. See cwd below how used.
-            decomp_k(c,j,i_met_lit) = (vmax_l1_m1 * m1_pool(c,j) / &
-                                       (km_l1_m1 + m1_pool(c,j))) * &
-                                      w_d_o_scalars
-            decomp_k(c,j,i_met_lit) = (vmax_l1_m2 * m2_pool(c,j) / &
-                                       (km_l1_m2 + m2_pool(c,j))) * &
+            ! TODO Excluded spinup terms for now (except see cwd below)
+            ! TODO Calc m*_conc = m*_pool(c,j) / dz(j) layer thickness
+            !           mgC/cm3 = gC/m3 * (1e3 mg/g) / (1e6 cm3/m3)
+            ! TODO decomp_k will end up in units of per hour and we
+            !      need per s so apply conversion to the coeffs above
+            decomp_k(c,j,i_met_lit) = (vmax_l1_m1 * m1_conc(c,j) / &
+                                       (km_l1_m1 + m1_conc(c,j)) + &
+                                       vmax_l1_m2 * m2_conc(c,j) / &
+                                       (km_l1_m2 + m2_conc(c,j))) * &
                                       w_d_o_scalars
 
-            decomp_k(c,j,i_str_lit) = (vmax_l2_m1 * m1_pool(c,j) / &
-                                       (km_l2_m1 + m1_pool(c,j))) * &
+            decomp_k(c,j,i_str_lit) = (vmax_l2_m1 * m1_conc(c,j) / &
+                                       (km_l2_m1 + m1_conc(c,j)) + &
+                                       vmax_l2_m2 * m2_conc(c,j) / &
+                                       (km_l2_m2 + m2_conc(c,j))) * &
                                       w_d_o_scalars
-            decomp_k(c,j,i_str_lit) = (vmax_l2_m2 * m2_pool(c,j) / &
-                                       (km_l2_m2 + m2_pool(c,j))) * &
+
+            decomp_k(c,j,i_avl_som) = (vmax_s1_m1 * m1_conc(c,j) / &
+                                       (km_s1_m1 + m1_conc(c,j)) + &
+                                       vmax_s1_m2 * m2_conc(c,j) / &
+                                       (km_s1_m2 + m2_conc(c,j))) * &
                                       w_d_o_scalars
 
-            decomp_k(c,j,i_pas_som) = (vmax_s3_m1 * m1_pool(c,j) / &
-                                       (km_s3_m1 + m1_pool(c,j))) * &
+! TODO Near line 1145 I see desorption for s3_s1
+            decomp_k(c,j,i_pas_som) = desorption * w_d_o_scalars
+! Controlled by microbial biomass through OXIDAT (line 1145 in testbed)
+! TODO OXIDAT = MICk * Vmax(K2) * SOMc / (KO(2) * Km(K2) + MICk) + &
+!               MICr * Vmax(R2) * SOMc / (KO(1) * Km(R2) + MICr)
+            decomp_k(c,j,i_slo_som) = TODO * w_d_o_scalars
+
+            decomp_k(c,j,i_cop_mic) = tau_m1 * m1_conc(c,j)**(densdep) * &
                                       w_d_o_scalars
-            decomp_k(c,j,i_pas_som) = (vmax_s3_m2 * m2_pool(c,j) / &
-                                       (km_s3_m2 + m2_pool(c,j))) * &
+            decomp_k(c,j,i_oli_mic) = tau_m2 * m2_conc(c,j)**(densdep) * &
```

Comment:
Does `m*_conc` come in here, or is the donor pool multiplied by microbial turnover when all the decomp_k values are used elsewhere?

Reply:
If we do leave `m1_conc` out here, I'm not sure how we'd be able to handle `densdep` here, which may be OK since we're setting it to 1 (at least for starters).

Reply:
I see what you mean. I was thrown off by the fact that we multiply all the others by `m*_conc` but the others are litter and soil, not microbes.

If we are keeping `densdep` as a placeholder, then I think the right answer is
```suggestion
            decomp_k(c,j,i_oli_mic) = tau_m2 * m2_conc(c,j)**(densdep - 1.0_r8) * &
```

Reply:
...to explain:
The minus 1 in `(densdep - 1)` just means that we are dividing the right hand side by m*_conc to account for the fact that later in the code `decomp_k` will get multiplied by `m*_conc`.